maths/volume.py

```diff
@@ -344,6 +348,7 @@ def vol_hollow_circular_cylinder(
         ...
     ValueError: outer_radius must be greater than inner_radius
     """
+    # Volume - (outer_radius squared - inner_radius squared) * pi * height
```

Comment:
Please move this comment down to be just above the calculation to make it easier for the reader to understand.